tests/props/host/host.c

```diff
@@ -172,7 +172,7 @@ int main(int argc, const char* argv[])
             is_signed,
             1111,                                        /* product_id */
             2222,                                        /* security_version */
-            OE_SGX_FLAGS_DEBUG | OE_SGX_FLAGS_MODE64BIT, /* attributes */
+            OE_SGX_FLAGS_DEBUG | OE_SGX_FLAGS_MODE64BIT | OE_SGX_FLAGS_KSS, /* attributes */
```

Comment:
Doesn't this break the tests? How is `OE_SGX_FLAGS_KSS` getting set on the enclaves?

Reply:
Yes actually. Will revert this one.